### ISSUE TITLE
Code not highlighted

### ISSUE THREAD

**Issue by pallix** [yjwen/org-reveal#151]
I'm using org-mode version 8.3.2 and latest org-reveal from MELPA, when the code snippset are exported, they are not highlighted. They are correctly highlighted in org-mode but not in the generated HTML.

**Comment by yjwen** [comment]
Please make sure htmlize.el is installed.

**Comment by pallix** [comment]
Oh sorry, I forgot to mention that but I have it installed!

**Comment by yjwen** [comment]
OK, Then could you please give me a sample org file and the exported HTML for a check?

**Comment by pallix** [comment]
Thanks for your help, documents are inlined below.


Input:

```
#+TITLE: Super Title
#+Author: P
#+Email: p@example.com

#+OPTIONS: reveal_center:t reveal_progress:t reveal_history:nil reveal_control:t
#+OPTIONS: reveal_mathjax:t reveal_rolling_links:t reveal_keyboard:t reveal_overview:t num:nil
#+OPTIONS: reveal_width:1200 reveal_height:800
#+OPTIONS: toc:1
#+REVEAL_ROOT: http://lab.hakim.se/reveal-js/
#+REVEAL_MARGIN: 0.1
#+REVEAL_MIN_SCALE: 0.5
#+REVEAL_MAX_SCALE: 2.5
#+REVEAL_TRANS: slide
#+REVEAL_THEME: moon
#+REVEAL_HLEVEL: 2
#+REVEAL_HEAD_PREAMBLE: <meta name="description" content="Adhocracy3 Backend">
#+REVEAL_PLUGINS: (highlight markdown notes)
#+REVEAL_EXTRA_CSS: ./local.css


# Reload org (C-u C-c C-x !) and try again.

* REST API
** Self-described
Self-described at /meta_api
#+ATTR_REVEAL: :frag roll-in
#+BEGIN_SRC javascript
  print('hello');
var x = 10;
    "resources": {
          "adhocracy_core.resources.rate.IRateVersion": 
          {
              "super_types": 
          [
              "adhocracy_core.interfaces.IItemVersion"
          ],
          "sheets": 
              [
                  "adhocracy_core.sheets.metadata.IMetadata",
                  "adhocracy_core.sheets.versions.IVersionable",
                  "adhocracy_core.sheets.rate.IRate"
              ]
          },
          "adhocracy_core.resources.geo.IMultiPolygon": ... 
#+END_SRC

```

Output:

```

<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<title>Super Title</title>
<meta name="author" content="(P)"/>
<link rel="stylesheet" href="http://lab.hakim.se/reveal-js/css/reveal.css"/>
<link rel="stylesheet" href="http://lab.hakim.se/reveal-js/css/theme/moon.css" id="theme"/>
<link rel="stylesheet" href="./local.css"/>
<!-- If the query includes 'print-pdf', include the PDF print sheet -->
<script>
    if( window.location.search.match( /print-pdf/gi ) ) {
        var link = document.createElement( 'link' );
        link.rel = 'stylesheet';
        link.type = 'text/css';
        link.href = 'http://lab.hakim.se/reveal-js/css/print/pdf.css';
        document.getElementsByTagName( 'head' )[0].appendChild( link );
    }
</script>
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<meta name="description" content="Adhocracy3 Backend">
</head>
<body>
<div class="reveal">
<div class="slides">
<section id="sec-title-slide">
<h1>Super Title</h1>
<h2>P</h2>
<h2><a href="mailto:p@example.com">p@example.com</a></h2>
<h2></h2>
</section>
<section id="table-of-contents">
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#/slide-orgheadline2">REST API</a></li>
</ul>
</div>
</div>
</section>



<section>
<section id="slide-orgheadline2">
<h2 id="orgheadline2">REST API</h2>
<div class="outline-text-2" id="text-orgheadline2">
</div></section>
</section>
<section>
<section id="slide-orgheadline1">
<h3 id="orgheadline1">Self-described</h3>
<p>
Self-described at /meta_api
</p>
<div class="org-src-container">

<pre class="fragment roll-in"><code class="javascript">  print('hello');
var x = 10;
    "resources": {
          "adhocracy_core.resources.rate.IRateVersion": 
          {
              "super_types": 
          [
              "adhocracy_core.interfaces.IItemVersion"
          ],
          "sheets": 
              [
                  "adhocracy_core.sheets.metadata.IMetadata",
                  "adhocracy_core.sheets.versions.IVersionable",
                  "adhocracy_core.sheets.rate.IRate"
              ]
          },
          "adhocracy_core.resources.geo.IMultiPolygon": ...
</code></pre>
</div>
</section>
</section>
</div>
</div>
<script src="http://lab.hakim.se/reveal-js/lib/js/head.min.js"></script>
<script src="http://lab.hakim.se/reveal-js/js/reveal.js"></script>

<script>
// Full list of configuration options available here:
// https://github.com/hakimel/reveal.js#configuration
Reveal.initialize({

controls: true,
progress: true,
history: false,
center: true,
slideNumber: true,
rollingLinks: true,
keyboard: true,
overview: true,
width: 1200,
height: 800,
margin: 0.10,
minScale: 0.50,
maxScale: 2.50,

theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
transition: Reveal.getQueryHash().transition || 'slide', // default/cube/page/concave/zoom/linear/fade/none
transitionSpeed: 'default',
multiplex: {
    secret: '', // null if client
    id: '', // id, obtained from socket.io server
    url: '' // Location of socket.io server
},

});
</script>
</body>
</html>

```

**Comment by yjwen** [comment]
Well, you are using highlight.js to highlight your codes. But unfortunately, highlight.js didn't work. I don't know what is wrong around the exported HTML and have filed an issue hakimel/reveal.js#1395 for a solution.

You can use org-reveal's highlighting by remove `highlight` from `#REVEAL_PLUGINS`, which use `htmlize.el` to export the highlighting scheme used in your emacs to the HTML.

**Comment by pallix** [comment]
Oh thanks, it's working now. I took the README.org of org-reveal as a sample, so I didn't notice that. Thank you.

**Comment by Titan-C** [comment]
What I see in this that when exporting none of the reveal_plugins are getting written in the output html file. There is nothing about the dependencies in the closing javascript block.

**Comment by Titan-C** [comment]
Sorry for spamming here, with the previous comment. I was upset since none of the plugins where loading(specially spaeker notes) but I realized is solved in #141 
I didn't have that the latest version but the one from MELPA, which although claims the same date, doesn't correspond to the master branch. Now plugins are loaded correctly.

To fix the issue with highlight.js you need to load the highlighting css file. Including in your org file

#+REVEAL_EXTRA_CSS: file:///path-to/reveal.js/lib/css/zenburn.css

fixes the problem. Zenburn.ccs comes by default in reveal.js, but can put any other css file from highlight.js

**Comment by yjwen** [comment]
Just merged master into stable branch to fix the reveal.js plug-in bug.
@Titan-C Yes. I realized some CSS is required for highlight.js to be work by the conversion of #140 .
I will fix the issue.
